### PR TITLE
Start gathering controller-manager profiles

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/bundle/test_metrics.go [hunk 1]
```diff
@@ -95,6 +107,9 @@ func (t *testMetrics) Execute(config *measurement.MeasurementConfig) ([]measurem
 	kubeSchedulerConfig := createConfig(config, map[string]interface{}{
 		"componentName": "kube-scheduler",
 	})
+	kubeSchedulerConfig := createConfig(config, map[string]interface{}{
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/common/bundle/test_metrics.go]: kubeControllerManagerConfig?

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/bundle/test_metrics.go]: done

File: clusterloader2/pkg/measurement/common/simple/profile.go [hunk 2]
```diff
@@ -44,6 +44,9 @@ type memoryProfileMeasurement struct{}
 
 // Execute gathers memory profile of a given component.
 func (c *memoryProfileMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	// TODO: Introduce two actions:
```
Comment (mm4tt) [inline clusterloader2/pkg/measurement/common/simple/profile.go]: NIT: Ownerless/bugless TODO, here and in other places.

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/simple/profile.go]: I'm working on it as we speak - that's why I didn't add it.

**Pull request merged**